frontend/src/pages/destructionlist/review/DestructionListReview.tsx

```diff
@@ -41,13 +48,81 @@ export function DestructionListReviewPage() {
     paginatedZaken,
     reviewItems,
     reviewResponse,
-    excludedZaakSelection,
   } = useLoaderData() as DestructionListReviewContext;
+
+  // Don't use the BaseListView zaak selection due to conflicting requirements, use custom implementation instead.
+  const [, handleSelect, { zaakSelectionOnPage, revalidateZaakSelection }] =
+    useZaakSelection<{
+      approved: boolean;
+      comment: string;
+    }>(
+      storageKey,
+      paginatedZaken.results,
+      filterSelectionZaken,
+      getSelectionDetail,
+      RestBackend,
+    );
+
+  // Poll for changes, update selection if a (remote) change has been made (by
+  // another reviewer).
+  usePoll(async () => {
+    const pollZaakSelection = await getZaakSelectionItems<{
+      approved: boolean;
+      comment: string;
+    }>(
+      storageKey,
+      paginatedZaken.results.map((z) => z.url as string),
+      true,
+      RestBackend,
+    );
+    const pollZaakUrls = Object.entries(pollZaakSelection)
+      .map(([url, { selected, detail }]) =>
+        [url, selected, detail?.approved].join(),
+      )
+      .sort();
+    const hookZaakUrls = Object.entries(zaakSelectionOnPage)
+      .filter(([, { selected }]) => selected)
+      .map(([url, { selected, detail }]) =>
+        [url, selected, detail?.approved].join(),
+      )
+      .sort();
```

Comment:
This is similar code as the _serializeZaakSelection earlier (plus sort)? maybe can be refactored to reuse the code?

Reply:
Yes, i think so. But i'm not entirely sure about the sort.